src/uproot/interpretation/library.py

```diff
@@ -617,7 +621,9 @@ def group(self, arrays, expression_context, how):
             return {_rename(name, c): arrays[name] for name, c in expression_context}
         elif how is None:
             if len(expression_context) == 0:
-                return awkward.Array(awkward.layout.RecordArray([], keys=[]))
+                return awkward.Array(
+                    awkward.contents.RecordArray([], keys=[])
+                )  # PLEASE SEE: RECORD ARRAY API seems to have changed
```

Comment:
Yes: "`keys`" is now named "`fields`" and it is no longer an optional argument.

```suggestion
                return awkward.Array(awkward.contents.RecordArray([], [])) 
```